src/Service/DublinCoreXMLGenerator.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+use App\Entity\Item;
+use Twig\Environment;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+class DublinCoreXMLGenerator
+{
+    public function __construct(
+        private Environment $twig,
+        private UrlGeneratorInterface $urlGenerator
+    )
+    {}
+
+    public function printWithTab(string $printString, int $tabCount)
+    {
+        $str = "";
+
+        for ($i = 0; $i < $tabCount*4; $i++)
+        {
+            $str .= " ";
+        }
+        
+        return $str.$printString;
+    }
+
+    public function generateDublinCoreXML(Item $item, string $reqDomain)
+    {
+        $filename = "item_xml.xml";
```

Comment:
I think that filename should be different for each item, e.g. sth like this: <item_name>_dc.xml.

Reply:
Done